### PR TITLE
update test cases

### PR DESCRIPTION
**Pull request by kfadillah**
Test cases moved/deleted:

URL: https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndReplaceClosureIfExistsFalse_entryNotExists,ID,,,

Situation: Test case got moved to AbstractStorageTest.java at commit 68d80efcd133d7ab3da71a586977304956df0856 and then renamed as AbstractPartitionStorageTest.java at commit f370441e239c70b6ef371b819232533c028f1f4e under the same directory.

NEW URL: https://github.com/apache/ignite-3,35eda6eee8601d1c34a1cf5c074b2fa169bb4a9a,modules/storage-api,org.apache.ignite.internal.storage.AbstractPartitionStorageTest.testGetAndReplaceClosureIfExistsFalse_entryNotExists,ID,,,

URL: https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.basic.TestSortedIndexMvStorageTest.testAbortRemove,ID,,,

Situation: Test file name is renamed as TestSortedIndexStorageTest at commit 188e63f2f62e51f2233554e917ad786f2b3418d4, and the testAbortRemove test is removed at commit 7e7ab4689db96ea83b11ec2233707645699e6683.

URL: https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.abortOfInsertMakesRowNonExistentForReadByTimestamp,ID,,,

Situation: Test case got moved to AbstractMvPartitionStorageTest.java under /modules/storage-api/src/test/java/org/apache/ignite/internal/storage/ at commit 1d76a6ee44fa37304b2c38cc1de677be966a63b0

New URL:https://github.com/apache/ignite-3,35eda6eee8601d1c34a1cf5c074b2fa169bb4a9a,modules/storage-api,org.apache.ignite.internal.storage.AbstractMvPartitionStorageTest.abortOfInsertMakesRowNonExistentForReadByTimestamp,ID,,,

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -1379,8 +1379,10 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/sql-engine,org.apache.ignite.internal.sql.engine.exec.MockedStructuresTest.testCreateTableMultiplePk,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/sql-engine,org.apache.ignite.internal.sql.engine.exec.MockedStructuresTest.testDropTable,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/sql-engine,org.apache.ignite.internal.sql.engine.StopCalciteModuleTest.testStopQueryOnNodeStop,ID,,,
+https://github.com/apache/ignite-3,35eda6eee8601d1c34a1cf5c074b2fa169bb4a9a,modules/storage-api,org.apache.ignite.internal.storage.AbstractMvPartitionStorageTest.abortOfInsertMakesRowNonExistentForReadByTimestamp,ID,,,
+https://github.com/apache/ignite-3,35eda6eee8601d1c34a1cf5c074b2fa169bb4a9a,modules/storage-api,org.apache.ignite.internal.storage.AbstractPartitionStorageTest.testGetAndReplaceClosureIfExistsFalse_entryNotExists,ID,,,
```
Comment (darko-marinov) [inline pr-data.csv:1383]: Thanks for putting all your changes in one commit. I've just noticed here the class name is `Abstract...Test`. Can you run this test by itself, e.g., `mvn test -Dtest=Abstract...Test#test...`. Many times one can't run an abstract class (or more precisely, a test in an abstract class), and has to run some concrete class that extends this abstract class. We want to provide here the name of the test that can be run, not the name where the test code may be written.

**Comment by kfadillah** [comment]
Update:
URL: https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndReplaceClosureIfExistsFalse_entryNotExists,ID,,,
Situation: Test case got moved to AbstractStorageTest.java at commit 68d80efcd133d7ab3da71a586977304956df0856 and then renamed as AbstractPartitionStorageTest.java at commit f370441e239c70b6ef371b819232533c028f1f4e under the same directory. However, there is no module that extends the abstract class.

Hence, no new URL is added.

URL: https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.abortOfInsertMakesRowNonExistentForReadByTimestamp,ID,,,

Situation: Test case got moved to AbstractMvPartitionStorageTest.java under /modules/storage-api/src/test/java/org/apache/ignite/internal/storage/ at commit 1d76a6ee44fa37304b2c38cc1de677be966a63b0, ran test with TestMvPartitionStorageTest which extends the abstract class, Nondex tool ran successfully.

Test success log: fa22-cs527-009.cs.illinois.edu:/home/kf14/igniteTestPass.log

File: pr-data.csv [hunk 2]
```diff
@@ -1395,7 +1395,7 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndRemoveClosure,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndRemoveClosure_failureBranch,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndReplaceClosureIfExistsFalse_entryExists,ID,,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndReplaceClosureIfExistsFalse_entryNotExists,ID,,,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndReplaceClosureIfExistsFalse_entryNotExists,ID,MovedOrRenamed,,https://github.com/apache/ignite-3/commit/f370441e239c70b6ef371b819232533c028f1f4e
```
Comment (darko-marinov) [inline pr-data.csv:1398]: The abstract test class where this test was moved seems to be extended with JUnit 5 annotation https://junit.org/junit5/docs/5.0.3/api/org/junit/jupiter/api/extension/ExtendWith.html (see https://github.com/apache/ignite-3/blob/56c217334884aac2ff4da6ca8023fa0d20a0028a/modules/storage-api/src/test/java/org/apache/ignite/internal/storage/AbstractPartitionStorageTest.java#L63).

Can you run the entire test suite for this module (just `mvn nondex:nondex`, without `-Dtest=...`) to see what tests get reported as flaky?

File: pr-data.csv [hunk 3]
```diff
@@ -1419,7 +1419,7 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
...
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.abortOfInsertMakesRowIdInvalidForAddWrite,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.abortOfInsertMakesRowIdInvalidForCommitWrite,ID,,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.abortOfInsertMakesRowNonExistentForReadByTimestamp,ID,,,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.abortOfInsertMakesRowNonExistentForReadByTimestamp,ID,MovedOrRenamed,,https://github.com/apache/ignite-3/commit/1d76a6ee44fa37304b2c38cc1de677be966a63b0
```
Comment (darko-marinov) [inline pr-data.csv:1422]: As above, see the comment for @ExtendWith.

**Review by darko-marinov — CHANGES_REQUESTED** [review]
The two tests that were moved are in the abstract classes that seem to be extended with `@ExtendWith`, so some of the tests may still be flaky.

You can split the PR into two: one to mark `Delete` and the other to later find what happened with these two `Moved[OrRenamed]` tests.

File: pr-data.csv (hunk 2 above)
Comment (kfadillah) [reply pr-data.csv:1398]: Thanks for the comment. I ran the nondex tool against the whole modules/api. I do not see any of the above cases failing. Please look into 
fa22-cs527-009.cs.illinois.edu:/home/kf14/ignite-3/modules/storage-api/.nondex/eCteOUpebXtV+FA7rkxhRbz8MC3Y6+EC9ikZ6fBrU= if needed. 

For other flaky tests within this modules/storage-api, there are 2 assertion error test cases in ConcurrentHashMapMvTableStorageTest (both written in AbstractMvTableStorageTest.java and not included in pr-data.csv), 5 assertion error test cases in DataStorageManagerTest (Already included in pr-data.csv). Also, there are 34 test error ("Failed to resolve parameter") in TestSortedIndexStorageTest. All test case failures are all included in the link above.

Comment (darko-marinov) [reply pr-data.csv:1398]: You need not list NetID publicly on GitHub; you can send private posts on Campuswire. For runs of NonDex, it's better to get a log, e.g., `mvn nondex:nondex |& tee log`? For new flaky tests that you see, you can add them later. Let me accept the current PR now.

**Pull request merged**